### PR TITLE
chore(css): Move CSS examples - masking

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- part of https://github.com/mdn/mdn/issues/597

This PR moves some CSS external examples into content as part of an initiative to reduce repo maintenance and code duplication.

### CODE REVIEW

File: files/en-us/web/css/mask-border/index.md [hunk 1]
```diff
@@ -72,11 +72,40 @@ mask-border: unset;
...
 
-![The image used for the mask examples on this page. The mask is a transparent square with three rows of three diamonds each. The diamonds are a very light, almost white, shade of grey. The middle part between the diamonds is also solid grey. The parts between the outside of the diamonds and the edge of the image are transparent.](mask-border-diamonds.png)
+<img src="https://mdn.github.io/shared-assets/images/examples/mask-border-diamonds.png" alt="The image used for the mask examples on this page. The mask is a transparent square with three rows of three diamonds each. The diamonds are a very light, almost white, shade of grey. The middle part between the diamonds is also solid grey. The parts between the outside of the diamonds and the edge of the image are transparent." data-flaw="image1" loading="lazy" style="background-color: black;">
 
 To match the size of a single diamond, we will use a value of 90 divided by 3, or `30`, for slicing the image into corner and edge regions. A repeat value of `round` will make the mask slices fit evenly, i.e., without clipping or gaps.
 
-{{EmbedGHLiveSample("css-examples/masking/mask-border.html", '100%', 800)}}
+```html
```
Comment (bsmth) [inline files/en-us/web/css/mask-border/index.md:79]:
````suggestion
```html live-sample___mask-border
````

**Review by bsmth — CHANGES_REQUESTED** [review]
Thanks, Onkar! Could we use the syntax `live-sample___sample-name`? This will make it a little easier to identify the source file (the EmbedGHLiveSample). For example: https://github.com/mdn/content/pull/36693/files

**Comment by OnkarRuikar** [comment]
> Thanks, Onkar! Could we use the syntax live-sample___sample-name?

The `mask-border` live sample identifier [didn't work](https://github.com/mdn/content/actions/runs/11795062842/job/32853997323?pr=36748#step:6:42). So I used `mask-border-example` format.

**Comment by bsmth** [comment]
> > Thanks, Onkar! Could we use the syntax live-sample___sample-name?
> 
> The `mask-border` live sample identifier [didn't work](https://github.com/mdn/content/actions/runs/11795062842/job/32853997323?pr=36748#step:6:42). So I used `mask-border-example` format.

that's strange, I wonder if there's a conflict with the file name? Anyway, thank you, I will take another look 👀

File: files/en-us/web/css/mask-border/index.md [hunk 2]
```diff
@@ -72,11 +72,40 @@ mask-border: unset;
 
 In this example, we will mask an element's border with a diamond pattern. The source for the mask is a ".png" file of 90 by 90 pixels, with three diamonds going vertically and horizontally:
 
-![The image used for the mask examples on this page. The mask is a transparent square with three rows of three diamonds each. The diamonds are a very light, almost white, shade of grey. The middle part between the diamonds is also solid grey. The parts between the outside of the diamonds and the edge of the image are transparent.](mask-border-diamonds.png)
+<img src="https://mdn.github.io/shared-assets/images/examples/mask-border-diamonds.png" alt="The image used for the mask examples on this page. The mask is a transparent square with three rows of three diamonds each. The diamonds are a very light, almost white, shade of grey. The middle part between the diamonds is also solid grey. The parts between the outside of the diamonds and the edge of the image are transparent." data-flaw="image1" loading="lazy" style="background-color: black;">
```
Comment (bsmth) [inline files/en-us/web/css/mask-border/index.md:75]: What's the data attribute for? 🤔

File: files/en-us/web/css/mask-clip/index.md [hunk 3]
```diff
@@ -80,9 +80,28 @@ One or more of the keyword values listed below, separated by commas.
 
 ### Clipping a mask to the border box
 
-Change the `mask-clip` value to any of the allowed values detailed above. If viewing the example in a Chromium-based browser change the value of `-webkit-mask-clip`.
+Open the live sample in the playground and change the `mask-clip` value to any of the allowed values detailed above.
```
Comment (bsmth) [inline files/en-us/web/css/mask-clip/index.md:83]:
```suggestion
Click "Play" in the live sample to open the code in the MDN Playground and change the `mask-clip` value to any of the allowed values described above.
```

Comment (bsmth) [reply files/en-us/web/css/mask-clip/index.md:83]: minor one to hint at clicking "Play"

File: files/en-us/web/css/mask-border/index.md (hunk 2 above)
Comment (OnkarRuikar) [reply files/en-us/web/css/mask-border/index.md:75]: It's yari internal mechanism we can remove it.
```suggestion
<img src="https://mdn.github.io/shared-assets/images/examples/mask-border-diamonds.png" alt="The image used for the mask examples on this page. The mask is a transparent square with three rows of three diamonds each. The diamonds are a very light, almost white, shade of grey. The middle part between the diamonds is also solid grey. The parts between the outside of the diamonds and the edge of the image are transparent." loading="lazy" style="background-color: black;">
```

Comment (bsmth) [reply files/en-us/web/css/mask-border/index.md:75]: righto 👍🏻

File: files/en-us/web/css/mask-origin/index.md [hunk 4]
```diff
@@ -74,9 +74,27 @@ One or more of the keyword values listed below, separated by commas.
 
 ### Setting mask origin to border-box
 
-Try some of the other possible values by updating the CSS in the box below.
+Open the live sample in in the playground and try some of the other possible `mask-origin` values.
```
[the remark this answers is not in the document]
Comment (OnkarRuikar) [reply files/en-us/web/css/mask-origin/index.md:77]:
```suggestion
Click "Play" in the live sample to open the code in the MDN Playground and try some of the other possible `mask-origin` values.
```

File: files/en-us/web/css/mask-composite/index.md [hunk 5]
```diff
@@ -53,7 +53,24 @@ For the composition the current mask layer is referred to as _source_, while all
...
 ### Compositing mask layers with addition
 
-{{EmbedGHLiveSample("css-examples/masking/mask-composite.html", '100%', 550)}}
+```html live-sample___mask-composite-example
+<div class="masked"></div>
+```
+
+```css live-sample___mask-composite-example
+.masked {
+  width: 100px;
```
Comment (bsmth) [inline files/en-us/web/css/mask-composite/index.md:62]: how does it look 100% wide?

File: files/en-us/web/css/mask-position/index.md [hunk 6]
```diff
@@ -57,10 +57,34 @@ One or more `<position>` values, separated by commas.
 
 ### Setting mask position
 
-Change the `mask-position` value to any of the allowed values detailed above.
-If viewing the example in a Chromium-based browser change the value of `-webkit-mask-position`.
+Open the live sample in the playground and change the `mask-position` value to any of the allowed values detailed above.
```
Comment (OnkarRuikar) [inline files/en-us/web/css/mask-position/index.md:60]:
```suggestion
Click "Play" in the live sample to open the code in the MDN Playground and change the `mask-position` value to any of the allowed values detailed above.
```

File: files/en-us/web/css/mask-image/index.md [hunk 7]
```diff
@@ -71,23 +71,24 @@ In the following cases, the mask is counted as a transparent black image layer:
...
 
   background: red;
-  mask-image: url(star.svg), radial-gradient(transparent 50%, black);
+  mask-image: url(https://mdn.github.io/shared-assets/images/examples/mask-star.svg),
+    radial-gradient(transparent 50%, black);
 }
 ```
 
-{{EmbedLiveSample("Setting a mask image with a URL", "100%", 250)}}
+{{EmbedLiveSample("mask-composite-example", "100%", 250)}}
```
Comment (bsmth) [inline files/en-us/web/css/mask-image/index.md:91]: typo I think:

```suggestion
{{EmbedLiveSample("mask-image-example", "100%", 250)}}
```

File: files/en-us/web/css/mask-composite/index.md (hunk 5 above)
Comment (bsmth) [reply files/en-us/web/css/mask-composite/index.md:62]: It's fine, feel free to ignore 👍🏻

**Review by bsmth — APPROVED** [review]
Looking great, one copy-paste error, but otherwise good to go! Thank you 👏🏻

File: files/en-us/web/css/mask-composite/index.md (hunk 5 above)
Comment (OnkarRuikar) [reply files/en-us/web/css/mask-composite/index.md:62]: It doesn't look good because masks are only 170px and 100px wide.

**Pull request merged**